crrExistingObjects.js

```diff
@@ -223,13 +240,45 @@ function _listObjectVersions(bucket, VersionIdMarker, KeyMarker, cb) {
         Prefix: TARGET_PREFIX,
         VersionIdMarker,
         KeyMarker,
-    }, cb);
+    }, (err, data) => {
+        if(err) {
+            // retry only on Network errors and if the process is set to retry
+            // on errors.
+            if (err.code && err.code === 'NetworkingError' && RETRY_ON_ERROR) {
```

Comment:
`err.code && err.code ===` is redundant.

Reply:
Ah, yes.